File: Server/ElectricBillPaymentSystem/ModelClass/DTO/BillTable.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ModelClass.DTO
{
    public class BillTable
    {
        public int BillId { get; set; }
        public int MeterReadingId { get; set; }
        public int CustomerId { get; set; }
        public int MeterId { get; set; }
        public double PreviousUnit { get; set; }
        public double CurrentUnit { get; set; }
        public double BillAmount { get; set; }
        public int CurrentMonth { get; set; }
        public int PreviousMonth { get; set; }
        public int BillStatus { get; set; }
        public string CreatedBy { get; set; }
        public DateTime CreatedDate { get; set; }
        public string UpdatedBy { get; set; }
        public DateTime? UpdatedDate { get; set; }
        public int Status { get; set; }
        public virtual MeterReadingTable MeterReadingTable { get; set; }
        public virtual Payment Payment { get; set; }
    }
}

File: Client/ElectricityBillManagement-Client/src/app/Model/BillTable.ts
import { MeterReadingTable } from "./MeterReadingTable";
import { Payment } from "./Payment";

export class BillTable{
    BillId: number;
    MeterReadingId: number;
    CustomerId: number;
    MeterId: number;
    PreviousUnit: number;
    CurrentUnit: number;
    BillAmount: number;
    CurrentMonth: number;
    PreviousMonth: number;
    BillStatus: number;
    CreatedBy: string;
    CreatedDate: Date;
    UpdatedBy: string;
    UpdatedDate: Date;
    Status: number;
    MeterReadingTable: MeterReadingTable;
    Payment: Payment;

}